components/partials/fee-discounts/overview.vue

```diff
@@ -219,6 +273,22 @@ export default Vue.extend({
       .finally(() => {
         this.status.setIdle()
       })
+  },
+
+  methods: {
+    handleScroll(e: Event) {
+      const target = e.target as HTMLElement
+      const offset = target.scrollLeft
+      const viewWidth = target.clientWidth
+      const total = target.scrollWidth - viewWidth
+      this.slideIndex = Math.round((offset / total) * 2)
```

Comment:
The blank row above. The reason why is that they are different blocks, the first block is defining variables and the second one is setting a state variable.